news/2018-12-31-beatmap-spotlights-fall-2018.md

```diff
@@ -0,0 +1,250 @@
+
+[Net0](https://osu.ppy.sh/users/5099768) beautifully portrays the story behind Utsu-P's "Uchuujin no I Love You" through his various strucutral and rhythm choices, leaving the player with a bizzare, yet unique play experience. His combo colors and storyboard visuals align together beautifully to help tell the story. That, combined with his various contrasts in intensity and use of repetitive slider shapes during the chorus shows a close attention to detail. If you think you have an eye for song representation and thought provoking maps, sit back and enjoy [Net0](https://osu.ppy.sh/users/5099768)'s Uchuujin no I Love You!
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/kimi-no-gin.jpg)](https://osu.ppy.sh/beatmapsets/876259)
+
+[Bearizm](https://osu.ppy.sh/users/3495259) kicks off the season with his return to the mapping scene after a year-long hiatus. His interpretation of Kalafina's "Kimi no Gin no Niwa" is one of the more expressive beatmaps in the game simply due to its use of dynamic slider velocity changes that follows the intensity of the vocals and violins, as well as having section emphasis, meaning that every chorus has a different set of rhythms and emphasis to show progression and a variety of different pacing and movements. This not only helps create variety throughout the song, but it compliments the dynamic slider velocity changes that the map offers because players will constantly need to pay full attention to momentum changes.
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/wakusei.jpg)](https://osu.ppy.sh/beatmapsets/841634)
+
+Marking an older mapper's return to the mapping scene, [gowww](https://osu.ppy.sh/users/119404)'s map of HACHI's "Suna no Wakusei" is nothing but a blast to play. His pattern-based mapping style and unconventional emphasis aren't intuitive for everyone, but engaging more with it will make this map stand out from the rest. The Extra difficulty keeps you on your toes throughout the whole map with a focus on different difficulty elements in each part, while finally climaxing into a straining jump section in the bridge with improvised emphasis. Along with well-made lower diffs that also differ from the norm, this mapset is a very unique and memorable experience.
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/osu/musique.jpg)](https://osu.ppy.sh/beatmapsets/816542)
+
+Fal's "ame no musique" is a super relaxing piece of music with a lot of rhythmic detail, and [LMT](https://osu.ppy.sh/users/7262798)'s rendition of it captures this feeling perfectly. The flow is smooth and the difficulty is fairly low, but the piano and string fills throughout the song are mapped in a very detailed way, making the rhythms throughout the map extremely varied and creating a unique challenge when played with difficulty increasing mods. Please try it our yourself!
```

Comment:
"throughout" is repeated, try another word such as "making the rhythms of/during the map"